pulsar-client-go/pulsar/consumer.go

```diff
@@ -77,6 +86,9 @@ type ConsumerOptions struct {
 	// Default is `Exclusive`
 	Type SubscriptionType
 
+	// Default is `Latest`
```

Comment:
Comment should indicate what is the purpose of setting a `SubscriptionInitPos`, eg: "If the subscription does not exist, it will be created either pointing at the end of the topic (`Latest`, which is the default) or on the earliest available message `Earliest`)"